test/Chainweb/Test/Pact/Utils.hs

```diff
@@ -274,3 +309,48 @@ withPactCtx v cutDB f
         evalPactServiceM ctx pact
   where
     start = testPactCtx v (someChainId v)
+
+initializeSQLite :: IO (IO (), SQLiteEnv)
+initializeSQLite = do
+      (file, del) <- newTempFile
+      -- TODO: Should change what is passed as the vfs_module.
+      e <- open_v2 (fromString file) (0x00000002 .|. 0x00000004 .|. 0x00010000) "unix"
```

Comment:
Needless to say, we can't be copying and pasting opaque values like this around.

Reply:
moved to utils, and documented

Reply:
:+1: at least name them